10-WrapUp/README.md

```diff
@@ -0,0 +1,3 @@
+### Wrap-Up
+
+**Author**:  Unknown
```

Comment:
Let's put Erik Tollerud & Kelle Cruz here.

Reply:
Done. Please merge if you are happy with this. 😸

Reply:
Looks great, thanks so much @pllim!